cli/bpmetadata/types.go

```diff
@@ -123,6 +172,91 @@ type BlueprintQuotaDetail struct {
 	QuotaType map[QuotaType]string `json:"quotaType" yaml:"quotaType"`
 }
 
+type BlueprintAuthor struct {
+	// Name of template author or organization.
+	Title string `yaml:"author"`
+
+	// Description of the author.
+	Description string `yaml:"author,omitempty"`
+
+	// Link to the author's website.
+	Url string `yaml:"url,omitempty"`
+}
+
+type SoftwareGroupType string
+
+const (
+	UnspecifiedSG SoftwareGroupType = "UNSPECIFIED"
+	Os            SoftwareGroupType = "OS"
```

Comment:
nit: acronyms should use all caps if exported, all lower if not exported
```suggestion
	OS            SoftwareGroupType = "OS"
```

Reply:
done